variables.sh

```diff
@@ -0,0 +1,38 @@
+# This should be included in both scripts, not invoked directly
+
+if [ -z "${VERSION:-}" ]; then
+  echo "Extracting version from tag ref ${YB_GIT_BRANCH:-}" 1>&2
+  VERSION="$(echo "${YB_GIT_BRANCH:-}" | sed -e 's|refs/tags/||g')"
+fi
+
+if [ -z "$VERSION" ]; then
+  echo "No version provided, won't release" 1>&2
+  exit 1
+fi
+
+if $(echo $VERSION | grep -vqo '^v'); then
```

Comment:
Using `$()` will execute the stdout, which I don't think is what we want here.

```suggestion
if echo "$VERSION" | grep -vqo '^v'; then
```

Reply:
Yes, that's what we want, `grep -q` doesn't print anything, it only returns a exit code

Reply:
I agree we want to run the grep, but `if` decides based on exit code. For example, this conditional will evaluate the _output_ of the echo as a command:

```bash
if $( echo /bin/false ); then
  echo "This won't be run"
fi
```

Whereas this uses the exit code of `echo` and thus will be run:

```bash
if echo /bin/false; then
  echo "This will be run"
fi
```

Reply:
You're right, sorry for the stubbornness :sweat_smile: